packages/material-ui/src/Tooltip/Tooltip.test.js

```diff
@@ -663,54 +663,6 @@ describe('<Tooltip />', () => {
     });
   });
 
-  describe('disabled button warning', () => {
-    it('should not raise a warning if title is empty', () => {
-      expect(() => {
-        render(
-          <Tooltip title="">
-            <button type="submit" disabled>
-              Hello World
-            </button>
-          </Tooltip>,
-        );
-      }).not.toErrorDev();
-
-      // TOD: Unclear why not running triggers microtasks but runAll does not trigger microtasks
-      // can be removed once Popper#update is sync
-      clock.runAll();
-    });
-
-    it('should raise a warning when we are uncontrolled and can not listen to events', () => {
-      expect(() => {
-        render(
-          <Tooltip title="Hello World">
-            <button type="submit" disabled>
```

Comment:
I don't understand. This still doesn't work. Why remove the warning?
https://codesandbox.io/s/simpletooltips-material-demo-forked-s1pqe?file=/demo.tsx

Reply:
I can't stress enough how useless these reviews are. Please describe the steps to reproduce, the expected behavior, the actual behavior and your environment. Same concern with your behavior as https://github.com/mui-org/material-ui/pull/23088#issuecomment-709982190

Reply:
I'm not reporting a bug. I'm providing a demonstration that the removed test case is still relevant.

- environment: Chrome, Edge
- steps to reproduce: hover the button
- expect/actual behavior, don't apply. I'm not reporting a bug.

The codesandbox renders the test case. It demonstrates that the tooltip doesn't display. It also demonstrates that the instruction in the documentation should be kept as they allow the tooltip to approach.

Does the PointerEvent API address this?